cookbooks/aws-parallelcluster-environment/resources/volume.rb

```diff
@@ -94,7 +94,14 @@
 
 action :unmount do
   shared_dir = format_directory(new_resource.shared_dir)
-
+  Chef::Log.info("Unmounting #{shared_dir} with the following")
```

Comment:
The log line should introduce the output of the next block, that is checking what processes are using the mountdir, rather than unmounting, right?

Reply:
yea, I was trying to be clever with this output, saying the active processes were present for the unmount, but I'll clarify